src/components/appFooter/AppFooter.js

```diff
@@ -7,14 +7,14 @@ const AppFooter = () => {
   return (
     <div className={cx('footer-container')}>
       <footer>
-{/*         India Mapped using{' '}
+        India Mapped using{' '}
         <a
           href="https://openstreetmap.org/"
           target="_blank"
           rel="noopener noreferrer"
         >
           OpenStreetMap.org
-        </a> */}
+        </a>
```

Comment:
This entire section can be left commented or filled with some other message. The attribution requirements to openstreemap are taken care of by plotly